iphone/Classes/TiUITabGroup.m

```diff
@@ -391,6 +391,40 @@ - (void)setActiveTabBackgroundImage_:(id)value
   controller.tabBar.selectionIndicatorImage = [self loadImage:value];
 }
 
+- (void)setTabsTransparent_:(id)value
+{
+  UITabBar *tabBar = [controller tabBar];
+
+  if ([TiUtils boolValue:value def:NO]) {
+    if (@available(iOS 13, *)) {
+      UITabBarAppearance *appearance = [tabBar.standardAppearance copy];
+
+      if ([TiUtils boolValue:value def:YES]) {
+        [appearance configureWithTransparentBackground];
+      } else {
+        [appearance configureWithDefaultBackground];
+      }
+
+      tabBar.standardAppearance = appearance;
+    } else {
+      UIGraphicsBeginImageContextWithOptions(CGSizeMake(1, 1), NO, 0.0);
+      UIImage *blank = UIGraphicsGetImageFromCurrentImageContext();
+      UIGraphicsEndImageContext();
+
+      tabBar.backgroundImage = blank;
+      tabBar.shadowImage = blank;
+    }
+  }
+}
+
+- (void)setTabsBackgroundView_:(id)value
+{
+  TiViewProxy *child = (TiViewProxy *)value;
+
+  [controller.tabBar addSubview:child.view];
```

Comment:
Just place the subview/child at the back to begin with to avoid the two calls?
```suggestion
  [controller.tabBar insertSubview:child.view atIndex:0];
```